### PR TITLE
Remove quotes from default data path

### LINKED ISSUE #6078: Synthetic data not building
https://github.com/Flowminder/FlowKit/blob/cb66267a2913116f85224fef54aceda62cc33e97/flowdb/testdata/bin/9900_ingest_synthetic_data.sh#L55

Quotes are causing the synthetic data build to fail

### PR DESCRIPTION
**Pull request by Thingus**
Closes #6078

### I have:

- [x] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [ ] Added tests for any new additions
- [ ] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [ ] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Description of what this PR _does_, and what it _changes_.

### CODE REVIEW

**Comment by jc-harrison** [comment]
I'm still puzzled as to why the `run_flowdbsynth_tests` CI job isn't failing due to this. If we can figure that out at this stage (and ideally change it so it _would_ fail if this happened again), that would be reassuring.

**Comment by Thingus** [comment]
I'm curious as to why this hasn't fixed the problem; logs from a clean gcloud instance using this branch attached

```

INSERT 0 1
INSERT 0 1
INSERT 0 1
CREATE INDEX
COMMIT
------------- // Done // ---------------
--2023-04-24 11:36:43--  https://data.biogeo.ucdavis.edu/data/gadm3.6/shp/gadm36_%22NPL%22_shp.zip
Resolving data.biogeo.ucdavis.edu (data.biogeo.ucdavis.edu)... 128.120.228.172
Connecting to data.biogeo.ucdavis.edu (data.biogeo.ucdavis.edu)|128.120.228.172|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-04-24 11:36:44 ERROR 404: Not Found.


PostgreSQL Database directory appears to contain a database; Skipping initialization

2023-04-24 11:36:45.444 UTC [1] LOG:  starting PostgreSQL 12.14 (Debian 12.14-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2023-04-24 11:36:45.445 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
```
Do we know which exact process interpolates the env variables? Is it Postgres, Docker or Bash?

**Comment by jc-harrison** [comment]
> I'm curious as to why this hasn't fixed the problem; logs from a clean gcloud instance using this branch attached
> 
> ```
> 
> INSERT 0 1
> INSERT 0 1
> INSERT 0 1
> CREATE INDEX
> COMMIT
> ------------- // Done // ---------------
> --2023-04-24 11:36:43--  https://data.biogeo.ucdavis.edu/data/gadm3.6/shp/gadm36_%22NPL%22_shp.zip
> Resolving data.biogeo.ucdavis.edu (data.biogeo.ucdavis.edu)... 128.120.228.172
> Connecting to data.biogeo.ucdavis.edu (data.biogeo.ucdavis.edu)|128.120.228.172|:443... connected.
> HTTP request sent, awaiting response... 404 Not Found
> 2023-04-24 11:36:44 ERROR 404: Not Found.
> 
> 
> PostgreSQL Database directory appears to contain a database; Skipping initialization
> 
> 2023-04-24 11:36:45.444 UTC [1] LOG:  starting PostgreSQL 12.14 (Debian 12.14-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
> 2023-04-24 11:36:45.445 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
> ```
> 
> Do we know which exact process interpolates the env variables? Is it Postgres, Docker or Bash?

Interesting... The script gets run by the entrypoint script in the Postgres docker image. I assume it's just run in the default shell within the container, but I haven't checked that.

Have you checked the '9900_ingest_synthetic_data.sh' script in the image you're running, to make sure the changes have actually made it into the image?

File: docker-compose-syntheticdata.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ services:
     image: flowminder/flowdb-synthetic-data:${CONTAINER_TAG:-latest}
     environment:
       SYNTHETIC_DATA_GENERATOR: ${SYNTHETIC_DATA_GENERATOR:?Must set SYNTHETIC_DATA_GENERATOR env var}
-      COUNTRY: ${COUNTRY:-"NPL"}
+      COUNTRY: ${COUNTRY:-NPL}
```
Comment (jc-harrison) [inline docker-compose-syntheticdata.yml:17]: I think the quotes also need to be removed from `DISASTER_REGION_PCOD` below.

**Review by jc-harrison — APPROVED** [review]

**Pull request merged**